docs/api-reference/io-columns-ranking.md

```diff
@@ -1,6 +1,5 @@
 ### Input and Output Columns
-The input label column data must be <xref:System.Single> and input group column
-data must be <xref:System.UInt32>. This trainer outputs the following columns:
+The input label column data must be <xref:System.Single> or [key](xref:Microsoft.ML.Data.KeyDataViewType) type, the feature column must be a known-sized vector of <xref:System.Single> and input row group column must be <xref:System.UInt32>. This trainer outputs the following columns:
 
```

Comment:
This line is very long and hard to review, could you break it up?
Also there seams to be a Uint32 still, could you change that to key....?

Reply:
Is that true for grioup id as well?

---
In reply to: [277191534](https://github.com/dotnet/machinelearning/pull/3455#discussion_r277191534) [](ancestors = 277191534)